specification/purview/data-plane/Azure.Analytics.Purview.MetadataPolicies/preview/2021-07-01/purviewMetadataPolicy.json

```diff
@@ -0,0 +1,613 @@
+          }
+        }
+      }
+    },
+    "/collections/{collectionName}/metadataPolicy": {
+      "get": {
+        "tags": [
+          "MetadataPolicy"
+        ],
+        "description": "Gets a metadata policy for collection",
+        "operationId": "MetadataPolicy_GetByCollection",
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "in": "path",
+            "name": "collectionName",
+            "description": "The name of an existing collection for which one policy needs to be fetched.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "$ref": "#/parameters/api-version"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success",
+            "schema": {
+              "$ref": "#/definitions/MetadataPolicy"
+            }
+          },
+          "default": {
+            "description": "An error response received from the Metadata Policy Service",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponseModel"
+            },
+            "headers": {
+              "x-ms-error": {
+                "type": "string",
+                "description": "The error code"
+              }
+            }
+          }
+        },
+        "x-ms-examples": {
+          "MetadataPolicy_GetByCollection": {
+            "$ref": "./examples/MetadataPolicy_GetByCollection.json"
+          }
+        }
+      }
+    },
+    "/metadataPolicies": {
+      "get": {
+        "tags": [
+          "MetadataPolicy"
+        ],
+        "description": "List metadata policies",
+        "operationId": "MetadataPolicy_ListAll",
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "#/parameters/api-version"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success",
+            "schema": {
+              "$ref": "#/definitions/MetadataPolicyList"
+            }
+          },
+          "default": {
+            "description": "An error response received from the Metadata Policy Service",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponseModel"
+            },
+            "headers": {
+              "x-ms-error": {
+                "type": "string",
+                "description": "The error code"
+              }
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        },
+        "x-ms-examples": {
+          "MetadataPolicy_ListAll": {
+            "$ref": "./examples/MetadataPolicy_ListAll.json"
+          }
+        }
+      }
+    },
+    "/metadataPolicies/{policyId}": {
+      "put": {
+        "tags": [
+          "MetadataPolicy"
+        ],
+        "description": "Updates a metadata policy",
+        "operationId": "MetadataPolicy_Update",
```

Comment:
Just to confirm, this only updates a metadata policy, it doesn't also create it?  If it does also create it, then the name should be `MetadataPolicy_CreateOrUpdate`.

Reply:
This only updates the policy. It can't be used to create.